### DISCUSSION TITLE
Need help to contribute in this project

### DISCUSSION THREAD

**Discussion by nobody-butcat** [ModECI/MDF#523] [Q&A]
Hello, I need some help in contributing to this project. I am at a beginner level in Github and this entire contribution thing. Any leads to quickly learn and get things started would be of immense help to me!
Also, if I am to contact any mentor of this project, which would be the right place? 

Please help me out a little.

**Comment by pgleeson** [comment]
Hi @nobody-butcat. Thanks for your interest in the project. Please see the discussion and the list of suggested tasks here: https://github.com/ModECI/MDF/discussions/509.

Go through the installation process (https://mdf.readthedocs.io/en/latest/api/Installation.html) in steps, googling for further information on anything you don't understand. Getting a local installation set up and running and understanding the existing examples is a minimum requirement for proceeding.

Please be aware though if you are interested in the Outreachy project that there has been a lot of interest already and some applicants have already made contributions to all of the suggested tasks. 

Good luck!
